packages/langium-cli/src/generator/module-generator.ts

```diff
@@ -21,10 +21,16 @@ export function generateModule(grammar: langium.Grammar, config: LangiumConfig):
     node.append(
         'import { ', grammar.name, "AstReflection } from './ast';", NL,
         'import { ', grammar.name, "GrammarAccess } from './grammar-access';", NL,
-        'import { ', grammar.name, "LanguageMetaData } from './meta-data';", NL,
         "import { Parser } from './parser';", NL, NL
     );
 
+    node.append('const metaData = {', NL);
+    node.indent(metaData => {
+        metaData.append(`languageId: '${config.languageId}',`, NL);
```

Comment:
What if this property is undefined? Shall we make it mandatory in the config?

Reply:
Yes #158 already makes it mandatory.